apps/ledger-live-mobile/src/components/PostOnboarding/PostOnboardingActionRow.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { useCallback } from "react";
+import { Flex, Icons, Tag, Text } from "@ledgerhq/native-ui";
+import { useTranslation } from "react-i18next";
+import { useNavigation } from "@react-navigation/native";
+import {
+  PostOnboardingActionState,
+  PostOnboardingAction,
+} from "@ledgerhq/types-live";
+import Touchable from "../Touchable";
+import { track } from "../../analytics";
+
+export type Props = PostOnboardingAction & PostOnboardingActionState;
+
+const PostOnboardingActionRow: React.FC<Props> = props => {
+  const {
+    navigationParams,
+    Icon,
+    title,
+    description,
+    tagLabel,
+    onStartEvent,
```

Comment:
`onStartEvent` maybe this name could be more explicit in the sense that it pertains to analytics. From the name I initially thought it was a callback function to be call on the start of "an event".